app/models/delete_queue_job.rb

```diff
@@ -133,6 +133,36 @@ def perform
     end
   end
 
```

Comment:
I'd rather see this in its own library called, e.g. 'UploadCleanupUtils' or 'ServerCleanupTasks' or somesuch, so that the job classes stay as single-purpose 'classes that perform the job in their name'

Reply:
Fair enough.

Reply:
4fb8abb